lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -188,8 +199,8 @@ function dosomething_mbp_install_productions() {
   $productions[4]['queues'][] = 'transactionalQueue';
   $productions[4]['queues'][] = 'loggingQueue';
   $productions[4]['queues'][] = 'activityStatsQueue';
-  $productions[4]['queues'][] = 'userCampaignActivityQueue';
```

Comment:
I think we still want this `userCampaignActivityQueue` right? 
that's the one that supports the mongo collection that stores the user activity stream